force-app/main/default/classes/ADDR_Addresses_UTIL_TEST.cls

```diff
@@ -37,8 +37,130 @@ private with sharing class ADDR_Addresses_UTIL_TEST {
      ****************************************************** isAddressEmpty ****************************************************
      **************************************************************************************************************************/
     @isTest
-    static void testIsAddressEmptyFalse() {
+    static void testIsAddressEmptyFalseMailingStreetNotEmpty() {
         List<Address__c> addresses = UTIL_UnitTestData_TEST.getMultipleTestAddresses(1);
```

Comment:
I know this might be a little silly here, but are we getting significant value from making an address record and then setting everything to null? Why not start with a blank address record if we're not performing DML.

Not a needed change now, but maybe something to look at in the future. I feel like `Address__c testAddress = new Address__c(MailingStreet__c = 'some random mailing street');` might be an easier read.

Reply:
That's a good point.